packages/core/src/sequelize-typescript.ts

```diff
@@ -569,13 +570,25 @@ export abstract class SequelizeTypeScript {
     return this.queryInterface.createSchema(schema, options);
   }
 
+  /**
+   * Alias of {@link AbstractQueryInterface#listSchemas}
+   *
+   * @param options
+   */
+  async listSchemas(options?: QiListSchemasOptions) {
+    return this.queryInterface.listSchemas(options);
+  }
+
   /**
    * Alias of {@link AbstractQueryInterface#showAllSchemas}
    *
+   * @deprecated Use {@link Sequelize#listSchemas} instead
```

Comment:
```suggestion
   * @deprecated Use {@link AbstractQueryInterface#listSchemas} instead
```

I think we want to reduce the amount of methods exposed in the Sequelize class and use QueryInterface instead (so you can remove `Sequelize#listSchemas`). Correct me if I'm wrong @ephys

Reply:
I've removed this now.